Library/Formula/nvi.rb

```diff
@@ -23,6 +23,12 @@ class Nvi < Formula
     sha256 "742c4578319ddc07b0b86482b4f2b86125026f200749e07c6d2ac67976204728"
   end
 
+  # support berkeley > 4, patch from debian
+  patch do
```

Comment:
Can you please submit this patch to the upstream developers of this project and add a link to the upstream patch submission and explanation of why the patch is needed in a comment in the formula file. Thanks!